src/utils.ts

```diff
@@ -256,15 +256,21 @@ export function calculateEditorColFromVimScreenCol(
     let currentCharIdx = 0;
```

Comment:
Can't we just use `charidx`, either as an async call or copying from the sources?